templates/kube-oidc-proxy.yaml

```diff
@@ -20,8 +20,8 @@ spec:
     - name: none
       enabled: true
   requires:
-    matchLabels:
-      kubeaddons.mesosphere.io/provides: ingresscontroller
+    - matchLabels:
```

Comment:
This also depends on dex and cert-manager

Reply:
done.